rqt_moveit/src/rqt_moveit/moveit_widget.py

```diff
@@ -34,29 +34,29 @@
 
 import os
 import sys
-from threading import Thread
-import time
+import threading
+import xmlrpclib
 
+from threading import Thread
```

Comment:
Please either import both class like this: `from threading import Event` and `from threading import Thread`) or use only `import threading` and then access them with `threading.Event` and `threading.Thread`.